frontend/src/lib/components/QuestionOptions.svelte

```diff
@@ -1,5 +1,8 @@
 <script lang="ts">
     import type { MultipleChoice } from '$lib/types/question';
+    import DisplayOption from './DisplayOption.svelte';
+    import { selected_store } from './store.js';
```

Comment:
You would need to reference the store in the `stores` folder like the comment I mentioned above.